src/api/appsettings.Development.json

```diff
@@ -7,5 +7,11 @@
   },
   "ConnectionStrings": {
     "BdmsContext": "Host=localhost;Username=SPAWNPLOW;Password=YELLOWSPATULA;Database=bdms"
+  },
+  "S3": {
+    "ACCESS_KEY": "REDSQUIRREL",
+    "BUCKET_NAME": "swissforages",
+    "ENDPOINT": "localhost:9000",
+    "SECRET_KEY": "YELLOWMONKEY"
```

Comment:
REQ: Da es dieselben Settings sind wie in `appsettings.json` müssen diese hier nicht nochmals aufgeführt werden.